bin/hdfcoinc/pycbc_combine_coincident_events

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+"""
+This program combines together a set of STATMAP files from disjoint times.
+The resulting file would contain triggers from the full set of input files
+"""
+import numpy
+import argparse
+import logging
+import h5py
+
+import lal
+
+import pycbc
+import pycbc.events
+import pycbc.version
+
+import h5py, numpy, argparse, logging, pycbc, pycbc.events, lal
+import pycbc.version
+
+def com(f, files, group):
+    """ Combine the same column from multiple files into another file f"""
+    try:
```

Comment:
better take more lines of code to explain what is being done, ie write out a loop over files explicitly, even if this makes the 'concatenate' operation clunkier.  at present the try-except is being applied to a complicated operation as a whole .. do we actually want to do try-except on every file separatel? and if the error happens in reading a specific file i think we'd like to print the filename.

Reply:
I removed the try except block. I think it is right to use list comprehension here.